xml/FrameworksIndex/net-5.0.xml

```diff
@@ -4414,51 +4384,6 @@
       <Member Id="M:System.Guid.TryParseExact(System.String,System.String,System.Guid@)" />
       <Member Id="M:System.Guid.TryWriteBytes(System.Span{System.Byte})" />
     </Type>
-    <Type Name="System.Half" Id="T:System.Half">
```

Comment:
is Half really being dropped?

Reply:
ping @carlossanlop @safern

Reply:
I don't think that is correct.

Reply:
This definitely shouldn't be getting dropped.

It is notably still in the ref assembly: <https://github.com/dotnet/runtime/blob/master/src/libraries/System.Runtime/ref/System.Runtime.cs#L2206>
nor was it flagged in the API diff: <https://github.com/dotnet/core/commit/851bb0cfc828a64ba63531f471a51f7c1d9e96ff>

Reply:
I'll take a look at what's wrong.

Reply:
Good catch, for some reason I got the Preview5 ref assemblies, I don't know what happened. Will send the new ones now.

Reply:
I've sent the updated ref assemblies.

Reply:
I'll close this PR, and start a new CI build.